##[pylyzer] failed /home/dmitry/progs/python/WISCourse/Python_assignments/day8/venv/lib/python3.10/site-packages/itsdangerous/exc.py 1736025949 3201
.___v_desugar_1 = pyimport "<failure>"
.<failure> = pyimport "<failure>"
.annotations: Never
.t = pyimport "<failure>"


.___v_desugar_2 = pyimport "<failure>"

.datetime: Never
.BadData: ClassType
.BadData <: global::Exception
.BadData.message: Str
.BadData.__call__: (message: Str) -> exc.BadData
.BadData.__str__: (self: exc.BadData) -> Str

.BadSignature: ClassType
exc = pyimport "exc"
.BadSignature <: .BadData
.BadSignature.payload: Never
.BadSignature.__call__: (message: Str, payload: Obj := Obj) -> exc.BadSignature

.BadTimeSignature: ClassType
.BadTimeSignature <: .BadSignature
.BadTimeSignature.date_signed: Never
.BadTimeSignature.__call__: (message: Str, payload: Obj := Obj, date_signed: Obj := Obj) -> exc.BadTimeSignature

.SignatureExpired: ClassType
.SignatureExpired <: .BadTimeSignature

.BadHeader: ClassType
.BadHeader <: .BadSignature
.BadHeader.original_error: NoneType or global::Exception
.BadHeader.header: Never
.BadHeader.__call__: (message: Str, payload: Obj := Obj, header: Obj := Obj, original_error: NoneType or global::Exception := NoneType or global::Exception) -> exc.BadHeader

.BadPayload: ClassType
.BadPayload <: .BadData
.BadPayload.original_error: NoneType or global::Exception
.BadPayload.__call__: (message: Str, original_error: NoneType or global::Exception := NoneType or global::Exception) -> exc.BadPayload

